nightingale_interface/screens/face_screen.py

```diff
@@ -0,0 +1,45 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+
+from kivymd.uix.label import MDLabel
+from kivy.uix.button import Button
+from kivy.uix.label import Label
+
+from kivy.uix.screenmanager import NoTransition
+from screens.screen_config import ScreenConfig as cfg
+
+
+class FaceScreen:
+    face_name = "facescreen"
+
+    def to_homescreen(self, button_data):
+        button_data.parent.manager.transition = NoTransition()
+        cfg.LAST_SCREEN = self.face_name
+        button_data.parent.manager.current = (
+            "homescreen"  # Should just be the "Home" screen
+        )
+
+    def face_build(self):
+        screen = Screen(name=self.face_name)
+
+        current_action_label = Label(
+            text=cfg.CURRENT_ACTION,
+            # font_style="H4",
+            halign="center",
+            pos_hint={"center_x": 0.5, "center_y": 0.9},
+        )
+        current_action_label.font_size = "50sp"
+
+        screen.add_widget(current_action_label)
+
+        # remove press color animation
+        screen.add_widget(
+            Button(
```

Comment:
Use the Image object instead of Button. Image also has the on_press and on_release parameters and also has additional features such as dynamic scaling

Reply:
I tried to do this(since you had it initially in the code) but the image is not able to be pressed even though the parameter is set to the callback. Is there possibly another function or object you were thinking about?

Reply:
discussed and not a priority for now. Need to make a class to do so https://stackoverflow.com/questions/33489143/kivy-image-as-button